elementary/utils/log.py

```diff
@@ -25,7 +28,30 @@ def format(self, record):
 
 
 FORMATTER = ColoredFormatter()
-LOG_FILE = "edr.log"
+LOG_FILE = os.getcwd() + "/edr_files/edr.log"
+
+
+class CustomerFileHandler(logging.FileHandler):
+    def emit(self, record):
+        file_path = os.getcwd() + "/edr_files"
+        try:
+            target_path_param = "target_path"
+            target_path_flag = "--target-path"
+            ctx = click.get_current_context()
+            ctx_args = ctx.args
+            if target_path_flag in ctx_args:
```

Comment:
Because in the context of the cli/cli.py it's an arg, but in the context of some other files it's in the params. I guess its based on if it's before or after the `get_cli_properties` function @haritamar